src/components/data/PathListAutomation.js

```diff
@@ -193,246 +199,292 @@ export default function PathListAutomation(props) {
+                                                            }
+                                                            required={true}
                                                         />
-                                                    </ListItem>
-                                                );
-                                            })}
-                                    </List>
-                                </Paper>
-                            </Grid>
-                            <Grid item xs>
-                                <Paper style={{ padding: theme.spacing(1) }}>
-                                    <Field
-                                        startingPath={startingPath}
-                                        name="dest"
-                                        id={buildId(
-                                            baseId,
-                                            ids.PATH_LIST_AUTO_DEST_FIELD
-                                        )}
-                                        acceptedType={ResourceTypes.FOLDER}
-                                        label={t("pathListDestLbl")}
-                                        component={InputSelector}
-                                        required={true}
-                                    />
-                                    <Field
-                                        id={buildId(
-                                            baseId,
+                                                        <Field
+                                                            id={buildId(
+                                                                baseId,
 
-                                            ids.PATH_LIST_AUTO_FILE_NAME_FIELD
-                                        )}
-                                        name="fileName"
-                                        required={true}
-                                        label={t("fileName")}
-                                        component={SaveAsField}
-                                    />
-                                </Paper>
-                            </Grid>
-                        </Grid>
-                        <Grid
-                            container
-                            direction="row"
-                            justify="flex-end"
-                            alignItems="flex-end"
-                            spacing={1}
-                        >
-                            {createPathListError && (
-                                <Grid item xs>
-                                    <ErrorTypographyWithDialog
-                                        baseId={baseId}
-                                        errorMessage={errorMsg}
-                                        errorObject={createPathListError}
-                                    />
-                                </Grid>
-                            )}
```

Comment:
Looks like this was accidentally removed?

Reply:
I have this in lines 247 - 256 but I'll move it to its previous location

Reply:
Ah, it's not that it needs to be moved - it's that it exists in 2 places intentionally, and you've removed 1 of the occurrences.